crates/medea_derives/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "medea-derives"
+version = "0.1.0-dev"
+edition = "2018"
+description = "Derive macros for medea."
+authors = ["Instrumentisto Team <developer@instrumentisto.com>"]
+homepage = "https://github.com/instrumentisto/medea"
+repository = "https://github.com/instrumentisto/medea"
+
+[dependencies]
+syn = "0.15.34"
+quote = "0.6.12"
```

Comment:
There is no reason to specify patch version. Specifying the minor version only (`syn = "0.15"`) is cleaner and more than enough.